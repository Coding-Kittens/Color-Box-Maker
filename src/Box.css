.Box{
  display: inline-block;
  border-radius: 15px;
margin: 7px;
margin-top: 10px;
}

.Box_button{
  position: relative;
  left: 10px;
  top: 10px;
}
