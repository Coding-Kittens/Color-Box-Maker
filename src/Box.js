import './Box.css'
const Box = ({backgroundColor,width,height,deleteBox,id})=>{
return <div className='Box' style={{height,width,backgroundColor}}>
<button className='Box_button' id={id} type="button" onClick={deleteBox} >X</button>
</div>
}
export default Box;
