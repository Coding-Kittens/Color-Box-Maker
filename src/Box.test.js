import {render} from '@testing-library/react';
import Box from './Box';

it('should renders without crashing',()=>{
  render(<Box/>);
})

it('should match snapshot',()=>{
const {asFragment} = render(<Box/>);
expect(asFragment()).toMatchSnapshot();
})
