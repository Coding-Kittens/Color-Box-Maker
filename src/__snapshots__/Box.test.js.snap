// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="Box"
  >
    <button
      class="Box_button"
      type="button"
    >
      X
    </button>
  </div>
</DocumentFragment>
`;
